lib/cdo/script_constants.rb

```diff
@@ -136,6 +136,7 @@ module ScriptConstants
       CSD4_2021_NAME = 'csd4-2021'.freeze,
       CSD5_2021_NAME = 'csd5-2021'.freeze,
       CSD6_2021_NAME = 'csd6-2021'.freeze,
+      CSD6_2021_NAME = 'csd7-2021'.freeze,
```

Comment:
Should be `CSD7_2021_NAME` right?

Reply:
Thank you for having a brain that is actually working today and catching all my mistakes :)